src/frontend/data/sideEffects/stopLive/index.spec.ts

```diff
@@ -0,0 +1,91 @@
+import fetchMock from 'fetch-mock';
+
+import { uploadState, liveState } from '../../../types/tracks';
+import { stopLive } from '.';
+
+jest.mock('../../appData', () => ({ appData: { jwt: 'some token' } }));
+
+describe('sideEffects/stopLive', () => {
+  afterEach(() => fetchMock.restore());
+
+  const video = {
```

Comment:
We are creating a complete and valid video object to avoid to cast it when we want to use it. Also it avoids regression, we are sure we are testing with a valid object with valid information